pages/dkp/konvoy/2.2/download/index.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: layout.pug
 navigationTitle: Download
-title: Download Konvoy
-excerpt: Learn how to download Konvoy
+title: Download DKP
+menuWeight: 2
```

Comment:
Don't know if this matters, but this pushes it above the Release notes in Konvoy